ParalayoutTests/InterpolationTests.swift

```diff
@@ -29,4 +31,248 @@ final class InterpolationTests: XCTestCase {
         XCTAssertEqual(actualRect, expectedRect)
     }
 
+    // MARK: - Float
+
+    func testLinearFloatInterpolation_fromMinToMax() {
```

Comment:
@NickEntin I've added some tests, these cover base cases, but do not cover every curve combination possible (especially for the function with a midpoint value). Let me know if this is good enough!

Reply:
These look fantastic! 🙌